src/ontology/slurp/ordo.tsv

```diff
@@ -1,118 +1,57 @@
 mondo_id	mondo_label	xref	xref_source	original_label	definition	parents
```

Comment:
@nicolevasilevsky This should be alright now after the bugfix for obsolete parents, but can you check.

Reply:
This looks really good to my eyes, well done!